vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -98,23 +102,56 @@ ent jhipster entity Conference
-    - `Overwrite src/main/resources/config/liquibase/master.xml?`
-    - `Overwrite src/main/java/com/mycompany/myapp/config/CacheConfiguration.java?`
 
-You have now generated an Entando project, including a Spring Boot microservice with database integration and React-based micro frontends.      
+You have now generated a Spring Boot microservice with database integration and React-based micro frontends.     
 
-### Project Structure
-   * ```/src/main/docker``` contains Docker files to help with local development environments.
+### Output
    * ```/src/main/java``` and ```src/main/resources``` contain the microservice codebase and configuration.
    * ```/ui``` holds the React-based micro frontends. By default, JHipster generates 3 MFEs per entity to contain the details, form, and table.
-   * ```/bundle``` is used to assemble the project code into an Entando Bundle.
+   * ```/src/main/docker``` contains Docker files to help with local development environments.
+
+Next you'll move the `/ui` and `/src/main/docker` files into the standard Entando bundle locations and then wire the MFEs and microservice together.
+
+## Finish configuring the bundle project
+1. Change back to the root directory of your project
+```shell
+cd ../..
+```
+2. Edit the `entando.json` and update `microservices/conference-ms` to set the healthCheckPath and dbms.
+```json
+   "healthCheckPath":"/management/health",
+   "dbms":"postgresql"
+```
+
+3. Move the generated `conference-table` MFE into the correct location in the bundle project. If you chose a different entity name, you'll need to adjust these commands accordingly.
+```shell
+ent bundle mfe add conference-table
+find microservices/conference-ms/ui/widgets/conference/tableWidget/. -mindepth 1 -maxdepth 1 -exec mv -t microfrontends/conference-table/ -- {} +
+```
+
+4. Now add an API claim to connect the `conference-table` MFE to the `conference-ms` microservice. This connection information is stored in the `entando-json`.
+```shell
+ent bundle api add conference-table conference-api --serviceName=conference-ms --serviceUrl=http://localhost:8081
+```
+
+5. Repeat the previous steps for the `conference-details` and `conference-form` MFEs.
+```shell
+ent bundle mfe add conference-details
+ent bundle mfe add conference-form
+find microservices/conference-ms/ui/widgets/conference/detailsWidget/. -mindepth 1 -maxdepth 1 -exec mv -t microfrontends/conference-details/ -- {} +
+find microservices/conference-ms/ui/widgets/conference/formWidget/. -mindepth 1 -maxdepth 1 -exec mv -t microfrontends/conference-form/ -- {} +
+ent bundle api add conference-details conference-api --serviceName=conference-ms --serviceUrl=http://localhost:8081
+ent bundle api add conference-form conference-api --serviceName=conference-ms --serviceUrl=http://localhost:8081
+```
+
+6. Finally, move the auxiliary services into place in the bundle project and enable the keycloak service for local tests.
```

Comment:
refine "into place" to be clarified/precise

Keycloak

punctuate with :

Reply:
Back-ticked keycloak since the service name is lowercase.